SAMtags.tex

```diff
@@ -234,13 +239,36 @@ \subsection{Barcodes}
 
 \begin{description}
 \item[BC:Z:\tagvalue{sequence}]
-Barcode sequence, with any quality scores stored in the {\tt QT} tag.
+Barcode sequence (Identifying the sample/library), with any quality scores stored in the {\tt QT} tag.
+The {\tt BC} tag should match the {\tt QT} tag in length. 
+What one does with multiple sample barcodes (e.g. one on each end of the template) is left to the implementation. 
+One current implementation concatenates all the barcodes and qualities and places a dash (`{\tt -}') between the barcodes from the same template. 
+
+\item[BX:Z:\tagvalue{sequence}]
+Corrected bases from the unique molecular identifier. unlike MI, this may be non-unique in the file. Should comprise of sequence base possibly separated by hyphens as {\tt RX}.
+
+\item[MI:Z:\tagvalue{str}]
+Molecular Identifier. A unique ID within the SAM file for the source molecule/cell from which this read is derived. All reads with the same MI tag represent the group of reads derived from the same source molecule/cell. 
```

Comment:
I'm not sure why "cell" found it's way in here.  A source molecule can only exist in one cell at a time, so I think it's redundant, but also confusing in that it could be interpreted as "for the source cell", which I don't think is right.